cactusbot/handler.py

```diff
@@ -6,10 +6,24 @@
 
 
 class Handlers(object):
-    """Handles all the events
+    """Evented controller for individual handlers.
 
-    For a function to have the ability to be used as an event handler, it must
+    For a method to have the ability to be used as an event handler, it must
     be prefixed with `on_`, and then followed by the event name.
+    This method gets a single argument of packet.
+
+    Packet can be the following types:
+
+    ========= ================
+    Type      Packet Type
+    ========= ================
+    `message` `MessagePacket`
+    `follow`  `EventPacket`
+    `sub`     `EventPacket`
+    `host`    `EventPacket`
+    ========= ================
```

Comment:
I'm not sure if this is all of them - those were just the ones I could think of off the top of my head. I'm pretty sure there are more, though... right?